holidays/countries/philippines.py

```diff
@@ -188,24 +189,6 @@ class PHL(Philippines):
     pass
 
 
-class PhilippinesChineseHolidays(_CustomChineseHolidays):
```

Comment:
Wouldn't it be better to keep it as list of verified dates? 👀

Reply:
`PhilippinesChineseHolidays` removal is now reverted in https://github.com/vacanza/holidays/pull/2332/commits/c37454606c0f54b30c057eb9d8db08a5c147b576, alongside a new 2025 entry 👍